src/traces/sankey/calc.js

```diff
@@ -14,13 +14,16 @@ var wrap = require('../../lib/gup').wrap;
 
 function circularityPresent(nodeList, sources, targets) {
 
-    var nodes = nodeList.map(function() {return [];});
+    var nodes = nodeList.map(function() { return []; });
```

Comment:
Would be nice to reuse:

https://github.com/plotly/plotly.js/blob/a132b85b899dd490216d12823be56ad3861f091f/src/lib/matrix.js#L13-L17